cbrkit/sim/collections.py

```diff
@@ -89,3 +89,167 @@ def wrapped_func(
         return dist2sim(distance)
 
     return wrapped_func
+
+
+def isolated_mapping(
+    element_similarity: SimPairFunc[Any, float],
+) -> SimPairFunc[Sequence[Any], float]:
```

Comment:
It would make sense  to use the typevar `cbrkit.typing.ValueType` instead of `Any` to ensure that the type of the similarity function matches the return type of this function.